script/hem.jl

```diff
@@ -0,0 +1,181 @@
+using Revise
+# using Combinatorics
+# using Ipopt
+using JuMP
+using YAML
+
+using HolisticElectricityModel
+import HolisticElectricityModelData
+
+const HEMDataRepo = HolisticElectricityModelData
+
+# Load config file
+if length(ARGS) > 0 # otherwise, define this variable in the REPL
+    config_fp = ARGS[0]
+else
+    config_fp = joinpath(@__DIR__, "configs", "hem_config.yaml")
+end
+config = YAML.load_file(config_fp)
+
+# File locations
+base_dir = abspath(joinpath(dirname(Base.find_package("HolisticElectricityModel")), ".."))
+hem_data_dir = dirname(dirname(Base.find_package("HolisticElectricityModelData")))
+input_path = joinpath(hem_data_dir, "inputs")
+
+include(joinpath(base_dir, "script", "config.jl"))
+include(joinpath(base_dir, "script", "config_data.jl"))
+
+# configure solver
+solver, = parse(config, "simulation_parameters", validators)
+@info "Running on environment $(splitpath(Base.active_project())[end-1]) with solver $(solver)"
+import_solver_package(solver)
+
+# ------------------------------------------------------------------------------
+# Input Folder and Data
+# ------------------------------------------------------------------------------
+
+(
+    balancing_areas,
+    base_year,
+    num_future_years,
+    num_ipps,
+    folder_suffix,
+    der_aggregator,
+    set_nuclear_varcost_negative,
+    sectors_with_county_level_load,
+    load_profiles_subdir,
+    skip_parse
+) = parse(config, "data_selection", validators)
+
+# Define scenario
+future_years = [base_year + i for i in range(1, num_future_years)]
+scenario = HEMDataRepo.DataSelection(balancing_areas, base_year, future_years, num_ipps)
+
+# Define parse options
+options = HEMDataRepo.ParseOptions(
+    der_aggregator,
+    set_nuclear_varcost_negative,
+    sectors_with_county_level_load;
+    load_profiles_subdir=load_profiles_subdir
+)
+
+bal_areas_len = length(balancing_areas)
+input_dir_name =
+    "ba_" *
+    "$bal_areas_len" *
+    "_base_" *
+    "$base_year" *
+    "_future_" *
+    "$num_future_years" *
+    "_ipps_" *
+    "$num_ipps" *
+    "_" *
+    "$folder_suffix"
+input_dir = joinpath(hem_data_dir, "runs", input_dir_name)
+
+if !skip_parse
+    mkpath(input_dir)
+    # For the following to work, someone needs to have done the following from HolisticElectricityModelData:
+    #     - In julia: run(output_dir = PROFILES_DIRECTORY, user = "nguo", hostname = HOSTNAME, dbname = DATABASE, port = PORT, pca_ids = nothing) to get residential and commercial profiles
+    #     - In command prompt: python inputs/write_industrial_profiles.py #ba to get industrial profiles
+    HEMDataRepo.parse_inputs(input_path, input_dir, scenario, options)
+end
```

Comment:
Let's simplify all of this to an `input_dir` and making sure that it exists

Reply:
Updated it and removed all dependencies to HolisticElectricityModelData.jl.